app/controllers/key_pairs_controller.rb

```diff
@@ -2,14 +2,25 @@ class KeyPairsController < ApplicationController
   # GET /key_pairs
   # GET /key_pairs.json
   def index
-    @key_pairs = KeyPair.all
+    # Grab the own user's keys only
+    @key_pairs = KeyPair.where("user_id = ?", @current_user.id)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.